src/App.jsx

```diff
@@ -14,45 +20,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getPreparedGoods(goods, { sortField, reverseField }) {
+  const visibleGoods = goods.map((good, index) => (
+    {
+      name: good,
+      id: (index + 1),
+      long: good.length,
+    }
+  ));
+
+  if (sortField) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_ALPH:
+          return good1.name.localeCompare(good2.name);
+
+        case SORT_FIELD_LENGTH:
+          return good1.long - good2.long;
+
+        default:
+          return '';
+      }
+    });
```

Comment:
let's use 'a' and 'b' instead of good1 and good2, using numbers in variable names is not the best practice, everyone knows that 'a' and 'b' are goods